### PR TITLE
weekly assets + gpt test

### PR DESCRIPTION
**Pull request by jxue16**

### CODE REVIEW

File: assets/anthropic.yaml [hunk 1]
```diff
@@ -570,3 +570,25 @@
...
   user_distribution: unknown
   failures: unknown
+- type: model
+  name: Claude 3
+  organization: Anthropic
+  description: The Claude 3 model family is a collection of models which sets new industry benchmarks across a wide range of cognitive tasks.
+  created_date: 2024-03-04
+  url: https://www.anthropic.com/news/claude-3-family
+  model_card: https://www-cdn.anthropic.com/de8ba9b01c9ab7cbabf5c33b80b7bbc618857627/Model_Card_Claude_3.pdf
+  modality: image, text; code, text
```
Comment (rishibommasani) [inline assets/anthropic.yaml:580]: remove code

File: assets/nvidia.yaml [hunk 2]
```diff
@@ -88,3 +88,25 @@
...
+  created_date: 2024-02-27
+  url: https://arxiv.org/pdf/2402.16819.pdf
+  model_card: none
+  modality: text; code, text
+  analysis: Evaluated on standard LLM benchmarks across a range of fields like reasoning, code generation, and mathematical skills.
+  size: 15B parameters (dense)
+  dependencies: []
+  training_emissions: unknown
+  training_time: 13 days
+  training_hardware: 384 DGX H100 nodes, each with 8 H100 80GB SXM5 GPUs
```
Comment (rishibommasani) [inline assets/nvidia.yaml:104]: maybe put the total number of gpus (i.e. what 384*8 is and then write this out after)

File: assets/samba.yaml [hunk 3]
```diff
@@ -0,0 +1,47 @@
...
+  description: SambaLingo is a suite of models that adapt Llama 2 to a diverse set of 9 languages.
+  created_date: 2024-02-26
+  url: https://sambanova.ai/blog/sambalingo-open-source-language-experts
+  model_card: 
+    explanation: The Arabic language model card is given, but there exist one for each of the other 8 languages in the collection.
+    value: https://huggingface.co/sambanovasystems/SambaLingo-Arabic-Base
+  modality: text; text
+  analysis: Evaluated on open source multilingual model benchmarks.
+  size: unknown
+  dependencies: [LLaMA 2]
```
Comment (rishibommasani) [inline assets/samba.yaml:14]: I think this should be typeset as Llama 2 everywhere?

**Review by rishibommasani — CHANGES_REQUESTED** [review]
Generally looks good to me, minor comments

File: assets/samba.yaml (hunk 3 above)
Comment (jxue16) [reply assets/samba.yaml:14]: Currently the original Llama is typeset as LLaMA everywhere, should I change those as well?

**Review by rishibommasani — APPROVED** [review]

**Pull request merged**